Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -292,6 +292,71 @@ function MiG_29A:defineBrakeLever(identifier, iCommand, arg_number, category, de
 	return control
 end
 
+-- Unit system set by the user settings, true if "metric", false if "imperial"
+local unit_metric = nil
+
+MiG_29A:addExportHook(function(dev0)
+	if unit_metric == nil then
+		local val = dev0:get_argument_value(1)
+
+		if val >= 0.01 then
+			unit_metric = false
+		else
+			unit_metric = true
+		end
+	end
+end)
+
+--- Defines a single gauge from a metric arg and it's limits and an imperial arg and it's limits based on user selected unit system
+--- @param identifier string the unique identifier for the control
+--- @param arg_number_metric integer the dcs argument number for the metric gauge
+--- @param arg_number_imperial integer the dcs argument number for the imperial gauge
+--- @param metric_limits number[] a length-2 array with the lower and upper bounds of the data for the metric control as used in dcs
+--- @param imperial_limits number[] a length-2 array with the lower and upper bounds of the data for the imperial control as used in dcs
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+function MiG_29A:defineMultiUnitFloatManualRange(identifier, arg_number_metric, arg_number_imperial, metric_limits, imperial_limits, category, description)
+	assert(#metric_limits == 2, string.format("%s may only contain a min and max value", identifier))
+	assert(#imperial_limits == 2, string.format("%s may only contain a min and max value", identifier))
+	local max_value = 65535
+	local alloc = self:allocateInt(max_value, identifier)
+
+	local function getArgFromUnit(dev0)
+		if unit_metric then
+			return dev0:get_argument_value(arg_number_metric)
+		else
+			return dev0:get_argument_value(arg_number_imperial)
+		end
```

Comment:
Might be slightly cleaner to have something like this?

```suggestion
local arg = unit_metric and arg_number_metric or arg_number_imperial
dev0:get_argument_value(arg)
```